README.rst

```diff
@@ -32,9 +32,10 @@ Install the latest Cookiecutter if you haven't installed it yet::
 
     pip install -U cookiecutter
 
-Generate your Invenio instance::
+You can choose the version of invenio you want to generate by selecting the corresponding branch
+when running cookiecutter. For example to generate a version 3.0 instance::
 
-    cookiecutter https://github.com/inveniosoftware/cookiecutter-invenio-instance.git
+    cookiecutter https://github.com/inveniosoftware/cookiecutter-invenio-instance@maint-3.0
```

Comment:
Does this work? If not something like this should: ``cookiecutter https://.…. -c maint-3.0``

Reply:
it doesn't, `-c maint-3.0` works indeed